.travis.yml

```diff
@@ -2,6 +2,10 @@ language: generic
 os: linux
 dist: focal
 
+branches:
+  except:
+    - github-actions
+
```

Comment:
I’m guessing that you’ll remove this before merging?

Reply:
> I’m guessing that you’ll remove this before merging?

Yes, this line was added to avoid triggering Travis.